ui/ui-tests/tests/submissions/literature.test.js

```diff
@@ -36,7 +36,10 @@ describe('Literature Submission', () => {
     await selectDocType(page, 'article');
 
     const image = await takeScreenShotForDesktop(page);
-    expect(image).toMatchImageSnapshot();
+    expect(image).toMatchImageSnapshot({
```

Comment:
why is this needed?

Reply:
it's flaky, has been feeling for a while here and there.